### PR TITLE
Another scaling take for HiDPI (I like this one best)

### PR DESCRIPTION
**Pull request by twa022**
#59 I think this better than what I posted on #61

### CODE REVIEW

File: usr/lib/mate-hud/mate-hud [hunk 1]
```diff
@@ -220,18 +220,33 @@ def get_menu(menuKeys):
...
-        xfce_custom_dpi = 0
-        try:
-            xfce_custom_dpi = Xfconf.Channel.new_with_property_base("xsettings", "/Xft").get_int("/DPI", 0)
-        except:
-            logging.debug('running XFCE, but xfconf gi repository not available')
-        if xfce_custom_dpi > 0:
-            dpi = xfce_custom_dpi
+    # from pyxr: https://github.com/WhatNodyn/Pyxr (MIT license)
+    # Get the xresource db so we can read Xft.dpi
+    res_prop = display.Display().screen().root.get_full_property(Xatom.RESOURCE_MANAGER, Xatom.STRING)
```
Comment (flexiondotorg) [inline usr/lib/mate-hud/mate-hud:227]: Sadly `pyxr` is not packaged for Debian/Ubuntu or Fedora/RHEL

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Can you rework this to not require pyxr? Perhaps this is available via GDK?

**Comment by twa022** [comment]
Updated to get DPI through GDK (poorly named function, but that's a lot easier...)

**Review by flexiondotorg — APPROVED** [review]
LGTM :+1:

**Pull request merged**